src/operator/numpy/np_ediff1d_op-inl.h

```diff
@@ -0,0 +1,224 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * \file np_ediff1d-inl.h
+ * \brief Function definition of numpy-compatible ediff1d operator
+ */
+
+#ifndef MXNET_OPERATOR_NUMPY_NP_EDIFF1D_OP_INL_H_
+#define MXNET_OPERATOR_NUMPY_NP_EDIFF1D_OP_INL_H_
+
+#include <mxnet/base.h>
+#include <mxnet/operator_util.h>
+#include <vector>
+#include "../mxnet_op.h"
+#include "../operator_common.h"
+#include "../elemwise_op_common.h"
+
+namespace mxnet {
+namespace op {
+
+struct EDiff1DParam : public dmlc::Parameter<EDiff1DParam> {
+  bool to_begin_arr_given, to_end_arr_given;
+  dmlc::optional<double> to_begin_scalar;
+  dmlc::optional<double> to_end_scalar;
+  DMLC_DECLARE_PARAMETER(EDiff1DParam) {
+    DMLC_DECLARE_FIELD(to_begin_arr_given).set_default(false).describe(
```

Comment:
pay attention to line wraps and indentations:
```c++
DMLC_DECLARE_FIELD(to_begin_arr_given)
  .set_default(false)
  .describe("blah blah")
```
same for all other parameter fields.